src/pages/apis/drug/remsspl/_infographics.yaml

```diff
@@ -0,0 +1,33 @@
+- title: "REMS SPL Submissions over time"
```

Comment:
@violetcrestedwren This chart pulls data only up until 20230621 for some reason.

Reply:
This reads the last updated date from downloads.json, which hasn't been run on the api server. This will work fine in prod.